src/map/Mapbox.ts

```diff
@@ -318,14 +318,12 @@ export default class Mapbox {
     }
 
     fitBounds(bbox: Bbox) {
-        if (bbox.every(num => num !== 0)) {
```

Comment:
Here we do not need the `bbox.every` check, because if there was no bounding box it would be undefined and since the type here is `Bbox` (not optional) we do not have to do this check and can rely on the fact it is not missing and we checked for the missing value elsewhere.

Reply:
Yes, as stated in the other comment